etl/grapher_model.py

```diff
@@ -224,9 +234,29 @@ class ChartRevisions(Base):
     updatedAt: Mapped[Optional[datetime]] = mapped_column(DateTime, init=False)
 
 
+class ChartConfig(Base):
+    __tablename__ = "chart_configs"
+    __table_args__ = (Index("idx_chart_configs_slug", "slug"),)
+
+    id: Mapped[bytes] = mapped_column(BINARY(16), primary_key=True, server_default=text("uuid_to_bin(uuid(),1)"))
+    uuid: Mapped[Optional[str]] = mapped_column(String(36), Computed("(bin_to_uuid(`id`,1))", persisted=False))
```

Comment:
We simplified this and just use a serialised uuid. The full definition of the `chart_configs` table is:

```
CREATE TABLE chart_configs (
    id char(36) NOT NULL PRIMARY KEY,
    patch json NOT NULL,
    full json NOT NULL,
    slug varchar(255) GENERATED ALWAYS AS (JSON_UNQUOTE(JSON_EXTRACT(full, '$.slug'))) STORED,
    createdAt datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updatedAt datetime DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
    INDEX idx_chart_configs_slug (slug)
)
```

So, removing `uuid` and changing the type of `id` to `String(36)` gets us there 👍🏻